server/command_create.go

```diff
@@ -170,29 +170,27 @@ func (p *Plugin) runCreateCommand(args []string, extra *model.CommandArgs) (*mod
 		license = config.E20License
 	}
 
-	if install.Version != "" || install.Image != "" {
-		err = validVersionOption(install.Version)
-		if err != nil {
-			return nil, true, err
-		}
+	err = validVersionOption(install.Version)
+	if err != nil {
+		return nil, true, err
+	}
 
-		var exists bool
+	var validTag bool
```

Comment:
I think we can remove this line as it was only needed before due to the if statement.